Architect-Coders/MU8
# Issue #62: Add Unit Tests to Events View Model

**What's in:**

- Created tests for EventsViewModel.
- Added missing test dependencies.
- Refactored scope to be send as parameter for every view model.

**Testing Done:**

* On emulator: Pixel 3a.

**How it looks:**

![image](https://user-images.githubusercontent.com/3532556/81228882-dc3f9700-8fbc-11ea-9938-f9f00f78bba8.png)

**How to test:**

- Launch the app and navigate through it. Verify is working normally.
- Go to the EventsViewModelTest and execute it, you should see all tests passing.

**Signed-off-by:**

* Diego Recalde <difereto89@gmail.com>
